docs/import/create_score_asset.js

```diff
@@ -220,13 +234,33 @@ function stringifyCollection(
   });
 }
 
+/**
+ * Sets the {@link geoidTag} for features in `censusTable`. Looks for both known
+ * geoid keys.
+ * @param {EeFC} censusTable Table downloaded from Census (income or SNAP)
+ * @return {ee.FeatureCollection} Processed collection
+ */
+function setGeoid(censusTable) {
+  return ee.FeatureCollection(censusTable).map((f) => {
+    // Starts with 150000000US, indicating block groups.
+    // https://www.census.gov/programs-surveys/geography/guidance/geo-identifiers.html
+    const geoid =
+        ee.String(getKeyOrBackup(f, censusGeoidKey, backupCensusGeoidKey));
+    return f.set(
+        geoidTag,
+        // If too short, null out: can happen with header rows.
```

Comment:
Sorry, not totally understanding the null out scenario (is it expected?) And how does this handle the old version?

Reply:
Expanded comment a bit. Better? Old version also has this geoid.